thoth/common/helpers.py

```diff
@@ -101,3 +106,29 @@ def get_service_account_token() -> str:
             "Unable to get service account token, please check "
             "that service has service account assigned with exposed token"
         ) from exc
+
+
+def to_camel_case(obj: Dict[str, Any]) -> Dict[str, Any]:
+    """Convert dictionary keys to camelCase."""
+    if isinstance(obj, dict):
```

Comment:
The function signature already states `obj` is a dictionary. It would be better to move this check down bellow before the actual recursion is done.

Reply:
I see your point. I do have to check for the dict at the beginning tho IMHO. However, I can refactor the function to contain an inner recursive function with a proper signature to make it a bit clearer.

Reply:
You can just change the signature to:

```python
def to_camel_case(obj: Any) -> Any:
```

Reply:
You can just change the signature to:

```python
def to_camel_case(obj: Any) -> Any:
```

Reply:
Aight, will do.. maybe even better to use a generic to emphasize that the return type is the same as the input `obj` type?

```python
def to_camel_case(obj: T) -> T:
```

Reply:
Sounds good :+1: